xpdacq/tests/test_beamtimeSetup.py

```diff
@@ -94,9 +94,9 @@ def test_start_beamtime(self):
         self.assertEqual(self.experimenters, self.bt.get('bt_experimenters'))
         self.assertEqual(self.wavelength, self.bt.get('bt_wavelength'))
         self.assertEqual(os.getcwd(), self.home_dir)
-        # FIXME
-        # strtScnLst = ['bt_bt.yml','ex_l-user.yml','sa_l-user.yml','sp_ct_0.1.yml','sp_ct_0.5.yml','sp_ct_1.yml','sp_ct_5.yml','sp_ct_10.yml','sp_ct_30.yml']
-        # self.assertEqual(newobjlist,strtScnLst)
+        # test prepoluate ScanPlan
+        self.assertEqual(len(self.bt.scanplans), 3)
+        self.assertEqual(self.bt.scanplans[0]['sp_args'], (5,))
```

Comment:
Would it be more functional to do this assert in a for loop?
```python
expected_sp_args = [5, 60, 1, 10, 30, 60]
for plan, expected in zip(self.bt.scanplans, expected_sp_args):
    self.assertEqual(plan['sp_args'], expected)
    # Maybe to a plumbing test running a prun of the scan plan?
```
This way if we add a new scan plan we just need to change the `sp_args` to include the updated scan plan arguments.